pkg/operator/operator.go

```diff
@@ -66,9 +71,19 @@ import (
 	"github.com/aws/karpenter-provider-aws/pkg/providers/version"
 )
 
+var BuildInfo = prometheus.NewGaugeVec(
+	prometheus.GaugeOpts{
+		Namespace: metrics.Namespace,
+		Name:      "build_info",
+		Help:      "A metric with a constant '1' value labeled by version from which karpenter was built.",
+	},
+	[]string{"version", "golang_version", "sha"})
```

Comment:
```suggestion
	[]string{"version", "goversion", "sha"})
```

Looking at this a little more closely, should we follow the prior art here that Prometheus has in terms of labels?

Reply:
Yep, you're right, missed that.
Thank you!